README.md

```diff
@@ -21,6 +18,65 @@ python -m pip install .
+
+1. In the `environment.sh` file, change the `DMD_ERA5_DVC_LOCAL_REMOTE` environment variable to your desired directory path
+2. Export the environment variables by running:
+
+    ```bash
+    source environment.sh
+    ```
+
+3. To perform the complete DVC setup, run:
+
+    ```bash
+    make dvc-setup
+    ```
+
+    Which will initialize DVC, set up the local remote, and set DVC auto-stage to `true`. If you don't want to set up the local remote, you can run:
+
+    ```bash
+    make dvc-init
+    make dvc-autostage
+    ```
+
+4. The previous command will have created a `.dvc` directory and a `.dvcignore` file. You will also see a few new files in the Git staging area. Commit these changes to Git with a message like "Set up DVC". Now you are ready to start using `dm-era5` with DVC.
+
+### Download ERA5 data
+
+Modify the `era5-download` section of the `config.ini` file to specify the desired download parameters. Then run:
+
+```bash
+python -m dmd_era5.era5_download.era5_download
+```
+
+Note that depending on the size of the download, this may be a memory-intensive process and you might need to run it on a HPC. The downloaded data will be saved in the `data/era5_download` directory as a NetCDF file, using the time range and time delta specified in `config.ini` as the file name (e.g. `2019-01-01T00_2019-01-02T00_1h.nc` for a time range from 2019-01-01 00:00 to 2019-01-02 00:00 with a time delta of 1 hour).
+
+If you followed the DVC setup instructions above, the downloaded data will be automatically tracked by DVC, and you will see three new files appearing in your Git staging area. For example, if you downloaded the file `2019-01-01T00_2019-01-02T00_1h.nc`, you will see the following files in the Git staging area:
+
+- `data/era5_download/.gitignore`, automatically created by DVC to tell Git to ignore the downloaded data file.
+- `data/era5_download/2019-01-01T00_2019-01-02T00_1h.nc.dvc`, automatically created by DVC to track the downloaded data version with a unique hash.
+- `data/era5_download/2019-01-01T00_2019-01-02T00_1h.nc.yaml`, which is a human-readable log of the different versions of the downloaded data, showing the metadata of each version. You can open this file to see the different versions of `2019-01-01T00_2019-01-02T00_1h.nc` that you have downloaded.
+
+You should now commit these files to Git with a message like "First download of 2019-01-01T00_2019-01-02T00_1h.nc". If you have set up a remote storage location for your data, you can push the data to the remote storage location by running:
+
+```bash
+dvc push
+```
+
+Once you have pushed to the remote, it is safe to delete the downloaded NetCDF file from `data/era5_download` if you wish, and you can even delete the DVC cache (which stores the data versions locally) by running `rm -rf .dvc/cache`. Note that deleting the data stored locally is an optional step that might be useful if you are running out of disk space.
```

Comment:
If DVC can find a suitable data file version in the remote, it will fetch it to the local cache, and then check it out to the working directory. So you don't need to manually copy anything back, it should be done automatically if you pushed the file to the remote. Deleting files locally (from the working directory or from the DVC cache) only makes sense if you are running low on disk space, otherwise you can leave them where they are. You should push to the remote regardless, to have a backup and so that your collaborators can pull the data if they wish to.

Reply:
I think I was anticipating the next step being 'run the model on the data you've just downloaded' so it was a little unexpected to remove it!

Reply:
Just wanted to highlight that the data is safely stored by DVC on the remote, so you don't need to worry. Will clarify instructions once I add the SVD step